docs/proposals/PROPOSAL_E.md

```diff
@@ -0,0 +1,138 @@
+# Proposal E - Platform Features
+
+This proposal leverages the previously reserved `features` field in the platform to give a finer grain of control to runtimes selecting an image to run.
```

Comment:
that's also the idea I mentioned in the proposal C for image selection: https://github.com/opencontainers/wg-image-compatibility/pull/8#discussion_r1456693674